src/main/java/de/thomas_oster/liblasercut/drivers/Ruida.java

```diff
@@ -180,45 +180,18 @@ public boolean canEstimateJobDuration()
     return false;
   }
 
-  /**
-   * 'runway' for laser to get up to speed when rastering (in mm)
-   *
-   */
+  // Kept to retain compatibility with older configs
+  @Deprecated
+  protected transient boolean useBidirectionalRastering = true;
   @Deprecated
   private transient Double rasterPadding = 0.;
-
-  @Override
-  public double getRasterPadding() {
-    return 0;
-  }
-
-  /*
-   * allow padding to move laser head outside of 'bed'
-   */
   @Deprecated
   private transient boolean allowOutsidePadding;
 
+  // fixed value of 0 because the sweep move adds all necessary padding
```

Comment:
```suggestion
  // fixed value of 0 because the device adds all necessary padding on its own (see writeJob(): processing mode ca01 01 bidirectional x-sweep)
```

Reply:
I don't think it necessary to repeat all of those details, grepping for "sweep" brings you there quickly.

If you want to add it, the name of the function would be `writeJobCode`. I think however everything after `:` could be left out.

Reply:
At least on first reading, "sweep move" was not clear to me because the move command itself is unchanged. I would suggest that we use the same words here as in writeJobCode(). You are right that the details should be left out here. How about

```suggestion
  // fixed value of 0 because the x-sweep processing mode adds all necessary padding. See writeJobCode().
```